modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/SidebarPanel.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayLoadingIndicator from '@clayui/loading-indicator';
+import React, {useImperativeHandle, useState} from 'react';
+
+import Sidebar from './components/Sidebar';
+
+const noop = () => {};
+
+const SidebarPanel = React.forwardRef(({onClose = noop}, ref) => {
+	const [isOpen, setIsOpen] = useState(true);
+
+	useImperativeHandle(ref, () => ({
+		close: () => {
+			setIsOpen(false);
+		},
+		open: () => {
+			setIsOpen(true);
+		},
+	}));
+
+	const onCloseHandle = () => (onClose ? onClose() : setIsOpen(false));
```

Comment:
I think this will always evaluate as true since we set a default value for `onClose`. Am I missing something there?

Also, I don't think the `(` `)` are necessary.

Reply:
> I think this will always evaluate as true since we set a default value for onClose. Am I missing something there?

You're totally right

> Also, I don't think the ( ) are necessary.

SF stuff